<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView"/>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="BlankWidget" name="blankWidget"/>
       <widget class="PaintWidget" name="paintWidget"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportComposition"/>
     <addaction name="actionExportModification"/>
    </widget>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="menuExport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreviousCell"/>
   <addaction name="actionNextCell"/>
   <addaction name="separator"/>
   <addaction name="actionComposeLayers"/>
   <addaction name="actionAutoTextRemovalAndBlankAlignment"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
   <addaction name="separator"/>
   <addaction name="actionPaintMask"/>
   <addaction name="actionEraseMask"/>
   <addaction name="actionEditBlank"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionDiscard"/>
  </widget>
  <action name="actionPreviousCell">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/circle-left.svg</normaloff>:/circle-left.svg</iconset>
   </property>
   <property name="text">
    <string>previous cell</string>
   </property>
   <property name="toolTip">
    <string>previous cell</string>
   </property>
  </action>
  <action name="actionNextCell">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/circle-right.svg</normaloff>:/circle-right.svg</iconset>
   </property>
   <property name="text">
    <string>next cell</string>
   </property>
   <property name="toolTip">
    <string>next cell</string>
   </property>
  </action>
  <action name="actionComposeLayers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/layer-group.svg</normaloff>:/layer-group.svg</iconset>
   </property>
   <property name="text">
    <string>compose layers</string>
   </property>
   <property name="toolTip">
    <string>compose layers</string>
   </property>
  </action>
  <action name="actionRemoveText">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/text-slash.svg</normaloff>:/text-slash.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Text</string>
   </property>
   <property name="toolTip">
    <string>Remove Text</string>
   </property>
  </action>
  <action name="actionEditBlank">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/border-all.svg</normaloff>:/border-all.svg</iconset>
   </property>
   <property name="text">
    <string>Edit Blank</string>
   </property>
   <property name="toolTip">
    <string>Edit Blank</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/magnifying-glass-plus.svg</normaloff>:/magnifying-glass-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/magnifying-glass-minus.svg</normaloff>:/magnifying-glass-minus.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionPaintMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/paint-brush-fine.svg</normaloff>:/paint-brush-fine.svg</iconset>
   </property>
   <property name="text">
    <string>Paint Mask</string>
   </property>
   <property name="toolTip">
    <string>paint mask</string>
   </property>
  </action>
  <action name="actionEraseMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/eraser.svg</normaloff>:/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>Erase Mask</string>
   </property>
   <property name="toolTip">
    <string>Erase Mask</string>
   </property>
  </action>
  <action name="actionDiscard">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/trash-xmark.svg</normaloff>:/trash-xmark.svg</iconset>
   </property>
   <property name="text">
    <string>Discard</string>
   </property>
   <property name="toolTip">
    <string>Discard</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/rotate-left.svg</normaloff>:/rotate-left.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/rotate-right.svg</normaloff>:/rotate-right.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAutoTextRemovalAndBlankAlignment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/wand-magic-sparkles.svg</normaloff>:/wand-magic-sparkles.svg</iconset>
   </property>
   <property name="text">
    <string>Auto Text Removal And Blank Alignment</string>
   </property>
   <property name="toolTip">
    <string>Auto Text Removal And Blank Alignment</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionall">
   <property name="text">
    <string>all</string>
   </property>
  </action>
  <action name="actionmodification_only">
   <property name="text">
    <string>modification only</string>
   </property>
  </action>
  <action name="actionExportComposition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>final composition</string>
   </property>
  </action>
  <action name="actionExportModification">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>modification only</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlankWidget</class>
   <extends>QWidget</extends>
   <header>blankwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header>paintwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
